xcode/Subconscious/Shared/Services/TranscludeService.swift

```diff
@@ -8,6 +8,11 @@
 import Foundation
 import Combine
 
+struct Transclusion {
+    let displayAddress: Slashlink
```

Comment:
`displayAddress` = The slashlink actually displayed in in the text
`address` = The absolute DID-peer Slashlink to the content

Open to naming tweaks as always.